lib/ssl/src/ssl_config.erl

```diff
@@ -2047,6 +2055,122 @@ cert_file_error(CertFile) ->
         end,
  {options, {certfile, binary_to_list(CertFile), Reason}}.
 
+validate_present_pem_files(Options, #{validate_pems := true}) ->
+    validate_cert_keys_pems(Options),
+    validate_cacerts_pem(Options),
+    validate_dh_pem(Options); %% Will only be present pr TLS-1.3
```

Comment:
nitpick: `pr TLS-1.3` we typically use PR for something else. I assume you've meant `prior`. Maybe avoid acronym in comment.

Reply:
I fixed that it was sort of a typo for pre, but I wrote prior.